geode-core/src/main/java/org/apache/geode/internal/cache/wan/serial/SerialGatewaySenderQueue.java

```diff
@@ -905,6 +915,10 @@ private void initializeRegion(AbstractGatewaySender sender, CacheListener listen
             new Object[] {this, this.regionName}),
             e);
       }
+      if ((this.region != null) && this.cleanQueues && this.enablePersistence) {
+        this.region.clear();
```

Comment:
Is this a distributed operation or only done locally?

Reply:
Locally.